app/models/publication.rb

```diff
@@ -28,7 +28,7 @@ class Publication < Publicationesque
   after_save :touch_statistics_announcement
   def touch_statistics_announcement
     if published? && !statistics_announcement.nil?
-      statistics_announcement.touch
+      statistics_announcement.reload.touch
```

Comment:
Is this deffo still needed? I seem to remember you saying it was to do with the way the associations were set up – just double checking it's still needed, because it doesn't seem 100% obvious why we should need to reload it.

Reply:
it is i'm afraid. The inverse relationship causes an eager load of the publication for the statistics announcement which doesn't capture the state transition that occurs from draft to published. 

I know it sucks 😢